.github/workflows/azure-static-web-apps-yellow-sea-0fa5a340f.yml

```diff
@@ -6,7 +6,7 @@ on:
     branches:
       - main
   pull_request:
-    types: [opened, synchronize, reopened, closed]
+    types: [closed]

```

Comment:
These changes were already mad in another PR, where are the doubled up? This could cause conflicts later

Reply:
I may be handling creating new branches incorrectly. I made created this branch before the previous comments were made and missed the correction on this branch. Is there an easier and more efficient way to go about this?